go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	github.com/krolaw/dhcp4 v0.0.0-20180925202202-7cead472c414
 	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.2.0
 	github.com/kubevirt/monitoring/pkg/metrics/parser v0.0.0-20230627123556-81a891d4462a
-	github.com/machadovilaca/operator-observability v0.0.20
+	github.com/machadovilaca/operator-observability v0.0.21-0.20240618140254-413f6b5035da
```

Comment:
don't forget to update to a released version

Reply:
if we need it (https://github.com/kubevirt/kubevirt/pull/12165#discussion_r1646148144) sure.